lib/whitehall_importer/integrity_checker/body_text_check.rb

```diff
@@ -2,19 +2,31 @@ module WhitehallImporter
   class IntegrityChecker::BodyTextCheck
     attr_reader :proposed_body_text, :publishing_api_body_text
 
+    ACCESSIBLE_FORMAT_NOTICE = / This file may not be suitable for users of assistive technology. (.*) It will help us if you say what assistive technology you use./.freeze
+
     def initialize(proposed_body_text, publishing_api_body_text)
       @proposed_body_text = proposed_body_text
       @publishing_api_body_text = publishing_api_body_text
     end
 
     def sufficiently_similar?
-      proposed_body = remove_attachment_file_size(proposed_body_text)
-      publishing_api_body = remove_attachment_file_size(publishing_api_body_text)
-      Sanitize.clean(publishing_api_body).squish == Sanitize.clean(proposed_body).squish
+      proposed_body = processed_body(proposed_body_text)
+      publishing_api_body = processed_body(publishing_api_body_text)
+
+      proposed_body == publishing_api_body
     end
 
   private
 
+    def processed_body(body_text)
+      processed_body = remove_attachment_file_size(body_text)
+      remove_accessible_format_notice(Sanitize.clean(processed_body).squish)
```

Comment:
I'd suggest we keep `Sanitize.clean` in `#sufficiently_similar?` and use it only after the body text has been processed.  This will help avoid possible unexpected behaviour that could occur if we sanitize the HTML midway through processing.

Reply:
@emmabeynon  we need to sanitize the text in order to do the `remove_accessible_format_notice` action as we need to remove the `html` elements and spacing etc for the check to work, so I am not sure how to keep it inside `sufficiently_similar?` without doubling up 

```
def sufficiently_similar?
      proposed_body = processed_body(proposed_body_text)
      publishing_api_body = processed_body(publishing_api_body_text)

      Sanitize.clean(proposed_body).squish == Sanitize.clean(publishing_api_body).squish
    end

  private

    def processed_body(body_text)
      processed_body = remove_accessible_format_notice(Sanitize.clean(body_text).squish)

      remove_attachment_file_size(processed_body)
    end
```

Reply:
^ test breaks due to `391 KB` vs ` 391KB` issue

Reply:
Ah ok that's fine, would you be able to update the commit message to include that reasoning?

Reply:
will do @emmabeynon. Also I ran this for each ID on the Trello card and all are ok in terms of the accessible format notice issue being fixed.

Reply:
Great news! 😄